packages/list/ListItem.tsx

```diff
@@ -79,9 +82,12 @@ export class ListItemBase<T extends HTMLElement = HTMLElement> extends React.Com
 
   componentDidMount() {
     this.initializeTabIndex();
+    if (this.props.initRipple) {
+      (this.props.initRipple as any)(this.listItemElement.current);
```

Comment:
Try updating React Ripple's initRipple type to:
```
  initRipple?: (surface: S | null, activator?: A | null) => void;
```

I don't think the React.Ref is necessary.

Reply:
I tried to remove `React.Ref`, but there are some code using that.
So I overwrite initRipple in props!